src/widgets/MaskedInputJqueryClientScript.php

```
@@ -0,0 +1,61 @@
+
+        if (is_array($widget->aliases) && $widget->aliases !== []) {
+            $js .= ucfirst($pluginName) . '.extendAliases(' . Json::htmlEncode($widget->aliases) . ');';
+        }
+
+        $id = $widget->options['id'] ?? '';
+
+        if (is_string($id) && $id !== '') {
+            $js .= 'jQuery("#' . $id . '").' . $pluginName . '(' . $widget->_hashVar . ');';
+        }
+
+        MaskedInputAsset::register($view);
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'yii\jquery\widgets\MaskedInputAsset' in method 'register'. (undefined)

(StaticAccess)